algebird-core/src/main/scala/com/twitter/algebird/Aggregator.scala

```diff
@@ -183,11 +183,21 @@ object Aggregator extends java.io.Serializable {
   def sortedTake[T: Ordering](count: Int): MonoidAggregator[T, PriorityQueue[T], Seq[T]] =
     new mutable.PriorityQueueToListAggregator[T](count)
   /**
+   * Same as sortedTake, but using a function that returns a value that has an Ordering.
+   */
+  def sortedTakeBy[T, U: Ordering](count: Int)(fn: T => U): MonoidAggregator[T, PriorityQueue[T], Seq[T]] =
```

Comment:
can we call this `sortByTake` mirroring the style of using scala names? This is like `sortBy(fn).take(count)`